pkg/ecdsa/dkg/params_pool_test.go

```diff
@@ -0,0 +1,176 @@
+package dkg
+
+import (
+	"github.com/keep-network/keep-core/pkg/internal/testutils"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/binance-chain/tss-lib/ecdsa/keygen"
+)
+
+func TestTSSPreParamsPool(t *testing.T) {
+	poolSize := 5
+
+	// Create new pool.
+	tssPool := newTestPool(poolSize)
+
+	if len(tssPool.pool) != 0 {
+		t.Errorf(
+			"invalid init length\nexpected: [%d]\nactual:   [%d]",
+			0,
+			len(tssPool.pool),
+		)
+	}
```

Comment:
We could reduce it to `testutils.AssertIntsEqual(t, "init length", 0, len(tssPool.pool))`. Same comment to all other integer assertions in this file.

Reply:
See https://github.com/keep-network/keep-core/pull/3112/commits/542385852b294c7a650b1a1033eb43861b2fc0e4